packages/story-utils/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "glob": "^7.1.6",
-    "lodash": "^4.17.20"
+    "lodash": "^4.17.20",
+    "zod": "^1.11.11"
```

Comment:
For validating JS, I'm partial to [Zod](https://github.com/colinhacks/zod) because of how well it ties in to Typescript (and it's designed to mirror Typescript behavior).

Reply:
Thanks for sharing, TIL